contracts/deposit-contract/depositContract.go

```diff
@@ -490,9 +568,9 @@ func (_DepositContract *DepositContractTransactorSession) Drain() (*types.Transa
 	return _DepositContract.Contract.Drain(&_DepositContract.TransactOpts)
 }
 
-// DepositContractChainStartIterator is returned from FilterChainStart and is used to iterate over the raw logs and unpacked data for ChainStart events raised by the DepositContract contract.
-type DepositContractChainStartIterator struct {
-	Event *DepositContractChainStart // Event containing the contract specifics and raw log
+// DepositContractDepositIterator is returned from FilterDeposit and is used to iterate over the raw logs and unpacked data for Deposit events raised by the DepositContract contract.
```

Comment:
```suggestion
// DepositContractDepositIterator is returned from FilterDeposit and is used to iterate over the raw logs and unpacked data for Deposit events raised by the deposit contract.
```

Reply:
@shayzluf this whole file is generated by abigen